ndk/src/main/jni/metadata.c

```diff
@@ -145,6 +145,25 @@ int bsg_get_map_value_int(JNIEnv *env, bsg_jni_cache *jni_cache, jobject map,
   return 0;
 }
 
+int bsg_populate_api_level_from_map(JNIEnv *env, bsg_jni_cache *jni_cache, jobject device_data) {
+    int value = -1;
```

Comment:
This could be initialised to `0` to be consistent with the `bsg_get_map_value_from_int` function above. But it's a massive edge case to not have an api level.